lotemplate/classes.py

```diff
@@ -77,6 +78,31 @@ def restart(self) -> None:
         self.__init__(self.host, self.port)
 
 
+class IfStatement:
+    """
+    Class representing an if statement in a template libreoffice
+    """
+    start_regex = r'\[\s*if\s*\$(\w+(\(.+?\))?)\s*(\=\=|\!\=)\s*([\w\-\.\,]+)\s*\]'
+    end_regex = r'(\[\s*endif\s*\])$'
+
+    def __init__(self, if_string):
+        self.if_string = if_string
+        match = re.search(self.__class__.start_regex, if_string, re.IGNORECASE)
+        self.variable_name = match.group(1)
```

Comment:
```py
self.variable_name, self.operator, self.value = match.groups()
```
This works only with the regex I provided. With the actual regex, it should be
```py
self.variable_name, _, self.operator, self.value = match.groups()
```
As one group is useless.

Reply:
fixed in the commit https://github.com/Probesys/lotemplate/commit/af7acc8853d78ff8ef1c0dc0d8ed1135638c9cef with the new regex